src/components/TextInput/BaseTextInput.js

```diff
@@ -304,7 +304,7 @@ class BaseTextInput extends Component {
                 {this.props.autoGrow && (
                     <Text
                         style={[...this.props.inputStyle, styles.hiddenElementOutsideOfWindow, styles.visibilityHidden]}
-                        onLayout={e => this.setState({textInputWidth: e.nativeEvent.layout.width})}
+                        onLayout={e => this.setState({textInputWidth: e.nativeEvent.layout.width + 1})}
```

Comment:
@aneequeahmad we gotta bump the width to `+ 2`. It worked better for me on cases like
- back from currency selection
- back from participant page

Also, we should def add a comment explaining why we are adding +2 to the width

https://user-images.githubusercontent.com/29673073/170892855-e6a72dd7-c016-4afa-9336-dcd77c4cc1be.mov

Reply:
Here's the storybook preview of the input after a `+2` width increase.

<img width="598" alt="image" src="https://user-images.githubusercontent.com/29673073/170893171-b03bba9a-a668-4f1e-9b77-15e4a79a9aac.png">